packages/libs/react-ui/src/components/Typography/Heading/Heading.css.ts

```diff
@@ -0,0 +1,131 @@
+/* eslint @typescript-eslint/naming-convention: 0, @kadena-dev/typedef-var: 0 */
```

Comment:
missing an index file

Reply:
Why do we need this? I prefer not globally disabling things for a whole file, but using `eslint-disable-next-line` or something similar. If this is a structural problem, we should modify the `@kadena-dev/eslint-config`

Reply:
I just opened another PR to update the @kadena-dev/eslint-config for typedef-var and am going to look into addressing the naming-convention one today 👍